code/datums/mutations/body.dm

```diff
@@ -375,3 +375,73 @@
 			owner.SetStun(owner.AmountStun()*2)
 			owner.visible_message("<span class='danger'>[owner] tries to stand up, but trips!</span>", "<span class='userdanger'>You trip over your own feet!</span>")
 			stun_cooldown = world.time + 300
+
+/datum/mutation/human/strongwings
+	name = "Strengthened Wings"
+	desc = "Subject's wing muscle volume rapidly increases."
+	quality = POSITIVE
+	difficulty = 12
+	instability = 15
+	species_allowed = list(SPECIES_APID, SPECIES_MOTH)
+
+/datum/mutation/human/strongwings/on_acquiring()
+	if(..())
+		return
+	var/obj/item/organ/wings/moth/mothwings
+	var/obj/item/organ/wings/bee/beewings = locate(/obj/item/organ/wings/bee) in owner.internal_organs
+	if(!beewings)
+		mothwings = locate(/obj/item/organ/wings/moth) in owner.internal_organs
+	if(beewings)
+		beewings.jumpdist += (6 * GET_MUTATION_POWER(src)) - 3
+		to_chat(owner, "<span class='notice'>Your wings feel stronger.</span>")
+	else if(mothwings)
+		mothwings.flight_level += 1
+		to_chat(owner, "<span class='notice'>Your wings feel stronger.</span>")
+		mothwings.Refresh(owner)
+	else
+		to_chat(owner, "<span class='warning'>You don't have wings to strengthen!</span>")
+
+/datum/mutation/human/strongwings/on_losing()
+	if(..())
+		return
+	var/obj/item/organ/wings/moth/mothwings
+	var/obj/item/organ/wings/bee/beewings = locate(/obj/item/organ/wings/bee) in owner.internal_organs
+	if(!beewings)
+		mothwings = locate(/obj/item/organ/wings/moth) in owner.internal_organs
+	if(beewings)
+		beewings.jumpdist -= (6 * GET_MUTATION_POWER(src)) - 3
+		to_chat(owner, "<span class='warning'>Your wings feel weak.</span>")
+	else if(mothwings)
+		mothwings.flight_level -= 1
+		to_chat(owner, "<span class='warning'>Your wings feel weak.</span>")
+		mothwings.Refresh(owner)
+	else
+		to_chat(owner, "<span class='warning'>The stubs of your wings feel weak!</span>")
+
+/datum/mutation/human/catclaws
+	name = "Cat Claws"
+	desc = "Subject's hands grow sharpened claws."
+	quality = POSITIVE
+	difficulty = 12
+	instability = 25
+	species_allowed = list(SPECIES_FELINID)
+	var/dmgadd = 6
```

Comment:
```suggestion
	var/dmgadd = 6
```
Can you name it something that will make sense when looking at the code 1 year later?